pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
Ah damn! Was thinking of this more at the API server level - sorry.

Metrics is what we're retrieving here so makes sense to use it to me. `/rawMetrics` & `/derivedMetrics` just don't read well to me. Would prefer something like `/metrics/raw` & /metrics/derived` personally, or a query param to select raw/derived,